### PR TITLE
Add command for removing tag pool permissions.

### PR DESCRIPTION
**Pull request by hodgestar**
Currently we don't have a command to do this.

### CODE REVIEW

**Comment by hodgestar** [comment]
Ready for review.

I ended up having to rework some bits of the command more than I might have liked, but the original structure was making things difficult.

File: go/base/management/commands/go_assign_tagpool.py [hunk 1]
```diff
@@ -8,69 +8,100 @@ class Command(BaseGoAccountCommand):
...
+            raise CommandError(
+                'Could not remove permission. Tagpool permission not found.')
+        if options['max_keys'] is not None:
+            raise CommandError(
+                'The max-keys option is may not be set when when deleting a'
+                ' permission.')
+        self.remove_permission(perms[0])
+
+    def remove_permission(self, tagpool_permission):
+        tagpool_permission.delete()
```
Comment (rudigiesler) [inline go/base/management/commands/go_assign_tagpool.py:85]: Why is this wrapped in a function?

**Comment by rudigiesler** [comment]
Just the one comment, otherwise looks good

File: go/base/management/commands/go_assign_tagpool.py (hunk 1 above)
Comment (jerith) [inline go/base/management/commands/go_assign_tagpool.py:85]: My guess would be for symmetry with `update_permission()` above?

**Comment by jerith** [comment]
:+1:

File: go/base/management/commands/go_assign_tagpool.py (hunk 1 above)
Comment (hodgestar) [inline go/base/management/commands/go_assign_tagpool.py:85]: Just to follow the pattern of function-for-processing-options and function-for-doing-work pattern that the other commands use.

**Comment by rudigiesler** [comment]
:+1: from me too

**Pull request merged**